src/app/public/modules/list-view-checklist/list-view-checklist.component.ts

```diff
@@ -93,13 +93,13 @@ export class SkyListViewChecklistComponent extends ListViewComponent implements
 
   @Input()
   public set selectMode(value: string) {
-    this._selectMode = value;
+    this._selectMode = value || 'multiple';
 
-    if (this._selectMode === 'single') {
+    if (this._selectMode === 'multiple') {
```

Comment:
Is the default value of `selectMode` "multiple"? If so, the pattern we follow in other `get/set` fields is to return the default in the `get`.
```
public set selectMode(value: string) {
  this._selectMode = value;

  if (this.selecteMode === 'multiple') {
  }
}

public get selectMode(): string {
  return this._selectMode || 'multiple';
}
```

Reply:
@Blackbaud-SteveBrush good tip. this is done.